test/forms/sections/consignee/ConsigneeExportEoriFormProviderSpec.scala

```diff
@@ -45,13 +45,15 @@ class ConsigneeExportEoriFormProviderSpec extends SpecBase with StringFieldBehav
       form,
       fieldName,
       maxLength,
-      FormError(fieldName, lengthKey, Seq(maxLength))
+      FormError(fieldName, lengthKey, Seq(maxLength)),
+      optPrefix = Some("GB")
```

Comment:
Instead of this, add explicit tests for the form for specific values. Similar to what Graham did here:
https://github.com/hmrc/emcs-tfe-frontend/pull/191/files

e.g.
```
"must transform the inputted ERN to uppercase" in {
      val result = form.bind(Map("ern" -> "gb00123456789", "entityGroup" -> "UK Record")).get
      result mustBe PrevalidateTraderModel("GB00123456789", EntityGroup.UKTrader)
    }

    "must transform the inputted ERN removing any spaces" in {
      val result = form.bind(Map("ern" -> "GB 0012 3456 789", "entityGroup" -> "UK Record")).get
      result mustBe PrevalidateTraderModel("GB00123456789", EntityGroup.UKTrader)
    }
```

Reply:
That also then tests the `replace` of the `transform` you've added to show it handles spaces too.